src/front-door/azext_front_door/azext_metadata.json

```diff
@@ -1,4 +1,4 @@
 {
     "azext.isPreview": true,
```

Comment:
If you plan to GA both front-door and waf-policy, then remove this line. If not, then please add the following in commands.py:
```Python
with self.command_group('network front-door', is_preview=True) as g:
  pass
```